sites/gcweb-menu/gcweb-menu-docs-fr.html

```diff
@@ -55,6 +55,9 @@ <h3>Classe CSS (v1.0)</h3>
 
 <h2>Historique</h2>
 
+<h3>Version 3.1</h3>
```

Comment:
This is a patch. there is no additional functionality available to our public API target audience (implementer and web author)

```suggestion
<h3>Version 3.0.1</h3>
```